.circleci/config.yml

```diff
@@ -216,48 +216,19 @@ jobs:
   docker-compose-test:
     executor: ubuntu-medium
     steps:
-      - checkout
-      - aws-setup
-      - aws-ecr-setup
-      - local-deploy-setup
-      - run: echo "export IMAGE_TAG=dev_$(git rev-parse --short=8 HEAD)" >> $BASH_ENV
-      # since we're running with `--build-all`, assume that if it passes, we have all images required for Forge
-      - run: aws ecr describe-images --repository-name="aptos/validator" --image-ids=imageTag=$IMAGE_TAG
-      - run:
-          name: Start local testnet docker-compose
-          shell: /bin/bash
-          working_directory: docker/compose/validator-testnet
-          command: |
-            export VALIDATOR_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/validator
-            export FAUCET_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/faucet
-            docker-compose up -d
-      - run:
-          name: Verify testnet docker-compose up
-          shell: /bin/bash
-          command: |
-            for i in $(seq 60); do
-              curl -s -w "\n%{http_code}\n" localhost:8080 | tee >(tail -1 > validator_ret.txt)
-              curl -s -w "\n%{http_code}\n" -X POST 'localhost:8000/mint?pub_key=459c77a38803bd53f3adee52703810e3a74fd7c46952c497e75afb0a7932586d&amount=20000000' \
-                | tee >(tail -1 > faucet_ret.txt)
-              if [ "$(cat validator_ret.txt)" = "200" ] && [ "$(cat faucet_ret.txt)" = "200" ]; then
-                echo "Both validator and faucet healthy"
-                exit 0
-              fi
-              sleep 1
-            done
-            exit 1
+      - docker-compose-setup
   sdk-typescript-test:
     executor: docker-typescript-builder
     steps:
-      - checkout
+      - docker-compose-setup
       # install packages for examples
       - run: cd ./ecosystem/typescript/sdk/examples/typescript && yarn install
       - run: cd ./ecosystem/typescript/sdk/examples/javascript && yarn install
       # Run package build+lint + tests
       - run: cd ./ecosystem/typescript/sdk && yarn install
       - run: cd ./ecosystem/typescript/sdk && yarn lint
       - run: cd ./ecosystem/typescript/sdk && yarn fmt:check
-      - run: cd ./ecosystem/typescript/sdk && yarn test
+      - run: cd ./ecosystem/typescript/sdk && APTOS_NODE_URL="http://localhost:8080" APTOS_FAUCET_URL="http://localhost:8000" yarn test
```

Comment:
nit: maybe set `APTOS_NODE_URL` and `APTOS_FAUCET_URL` in the `docker-compose-setup` command?